proxy/certs/certs.go

```diff
@@ -32,6 +32,7 @@ import (
 	"github.com/GoogleCloudPlatform/cloudsql-proxy/proxy/util"
 	"google.golang.org/api/googleapi"
 	sqladmin "google.golang.org/api/sqladmin/v1beta4"
+	"strings"
```

Comment:
yet more nitpicking: import this in the top block of imports please.

Reply:
Sure. Thanks for all these comments. They are really helpful!